pairtools/pairtools_stats.py

```diff
@@ -25,21 +27,48 @@
     " all overlapping statistics. Non-overlapping statistics are appended to"
     " the end of the file.",
 )
+@click.option(
+    "--analyse-bytile-dups",
```

Comment:
To my understanding, this is a flag. Why is it string then? 
I am also a big fan of recent click flag arguments, where both options can be specified explicitly: 
https://click.palletsprojects.com/en/7.x/options/#boolean-flags

Should we use it here as well? 

Maybe also simplify the name of this version:
--bytile-dups/--no-bytile-dups

What do you think?